src/state/apis/snapshot/selectors.ts

```diff
@@ -13,5 +15,10 @@ export const selectVotingPowerByModel = (state: ReduxState) => state.snapshot.vo
 export const selectVotingPower = createSelector(
   selectVotingPowerByModel,
   selectFeeModelParamFromFilter,
-  (votingPowerByModel, feeModel) => votingPowerByModel[feeModel!],
+  selectAccountIdsByChainId,
+  (votingPowerByModel, feeModel, accountIdsbyChainId) => {
+    const ethAccountIds = accountIdsbyChainId[ethChainId]
+    if (!ethAccountIds?.length) return undefined
```

Comment:
To honor `isFallbackFees` in `calculateFees` if we want to do anything fancy for fallback fees in particular - else, just return `'0'` and let's handle this gracefully for whatever fees would be for 0 FOX vp 

https://github.com/shapeshift/web/blob/0bc5963d5cdbeebf942b43b79756030660f5c072/src/lib/fees/model.ts#L58-L59

Reply:
Confirmed with @twblack88 we just assume 0 vp fees, getting the "max" (i.e for 0 held FOX) fees